modules/110-istio/templates/cni/daemonset.yaml

```diff
@@ -0,0 +1,192 @@
+metadata:
+  name: istio-cni-node
+  namespace: d8-{{ .Chart.Name }}
+  {{- include "helm_lib_module_labels" (list $ (dict "app" "istio-cni-node")) | nindent 2 }}
+spec:
+  revisionHistoryLimit: 2
+  selector:
+    matchLabels:
+      app: istio-cni-node
+  updateStrategy:
+    rollingUpdate:
+      maxSurge: 0
+      maxUnavailable: 1
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: istio-cni-node
+        sidecar.istio.io/inject: "false"
+    spec:
+      imagePullSecrets:
+      - name: deckhouse-registry
+      containers:
+        - name: install-cni
+          args:
+            - --log_output_level=default:info
+          command:
+            - install-cni
+          {{- include "helm_lib_module_pod_security_context_run_as_user_root" . | nindent 10 }}
+          env:
+            - name: CNI_NETWORK_CONFIG
+              valueFrom:
+                configMapKeyRef:
+                  key: cni_network_config
+                  name: istio-cni-config
+            - name: CNI_NET_DIR
+              value: /etc/cni/net.d
+            - name: CHAINED_CNI_PLUGIN
+              value: "true"
+            - name: REPAIR_ENABLED
+              value: "true"
+            - name: REPAIR_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: spec.nodeName
+            - name: REPAIR_LABEL_PODS
+              value: "true"
+            - name: REPAIR_DELETE_PODS
+              value: "true"
+            - name: REPAIR_RUN_AS_DAEMON
+              value: "true"
+            - name: REPAIR_SIDECAR_ANNOTATION
+              value: sidecar.istio.io/status
+            - name: REPAIR_INIT_CONTAINER_NAME
+              value: istio-validation
+            - name: REPAIR_BROKEN_POD_LABEL_KEY
+              value: cni.istio.io/uninitialized
+            - name: REPAIR_BROKEN_POD_LABEL_VALUE
+              value: "true"
+            - name: NODE_NAME
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: spec.nodeName
+            - name: LOG_LEVEL
+              value: debug
+            - name: GOMEMLIMIT
+              valueFrom:
+                resourceFieldRef:
+                  divisor: "0"
+                  resource: limits.memory
+            - name: GOMAXPROCS
+              valueFrom:
+                resourceFieldRef:
+                  divisor: "0"
+                  resource: limits.cpu
+          image: {{ include "helm_lib_module_image" (list $ (printf "cni%s" $imageSuffix)) }}
+          imagePullPolicy: IfNotPresent
+          readinessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /readyz
+              port: 8000
+              scheme: HTTP
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 1
+          resources:
+            requests:
+              cpu: 100m
+              memory: 100Mi
```

Comment:
Resources requests should not be hardcoded. The clause is missing.

Reply:
Ref: https://github.com/deckhouse/deckhouse/blob/main/modules/470-chrony/templates/daemonset.yaml#L1